src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -751,22 +747,16 @@ protected StatementSyntax GetStatementContainingInvocationToExtractedMethodWorke
             {
                 // Only need to update for nullable reference types in return
                 if (methodSymbolResult.Data.ReturnType.NullableAnnotation != NullableAnnotation.Annotated)
-                {
-                    return await base.UpdateMethodAfterGenerationAsync(originalDocument, methodSymbolResult, cancellationToken).ConfigureAwait(false);
-                }
+                    return originalDocument;
```

Comment:
this is all the base method did.  I made the base method abstract, and just inlined it to teh use sites in teh override method.